StackUsingLinkedList.cpp

```diff
@@ -0,0 +1,80 @@
+#include<stdio.h>
+#include <stdlib.h>
+struct node {
```

Comment:
Please change your code formatting. [See here](https://google.github.io/styleguide/cppguide.html) and https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace